### PR TITLE
Gtranslator 3.30.1 release

### PR DESCRIPTION
**Pull request by danigm**
I've just made a new release of Gtranslator, we can update the flatpak to the new version.

### CODE REVIEW

**Comment by jurf** [comment]
bot, build

File: org.gnome.Gtranslator.yml [hunk 1]
```diff
@@ -1,5 +1,4 @@
 id: org.gnome.Gtranslator
-branch: stable
 runtime: org.gnome.Platform
 runtime-version: 3.30
```
Comment (TingPing) [inline org.gnome.Gtranslator.yml]:
```suggestion
runtime-version: '3.30'
```

File: org.gnome.Gtranslator.yml [hunk 2]
```diff
@@ -11,9 +10,11 @@ build-options:
 cleanup:
   - '/include'
```
[the remark this answers is not in the document]
Comment (danigm) [reply org.gnome.Gtranslator.yml]: I think it's okay as is, to keep all entries in the list with the same format.

**Comment by nedrichards** [comment]
bot, build

**Comment by nedrichards** [comment]
Builds fine, if you know it works in testing then looks good to merge. @danigm since you're part of upstream I sent you an invtiation to this repo so you should be able to merge, kick off builds etc.

**Comment by danigm** [comment]
I didn't apply this merge request yet because there was an error in the localization of the app. This has been resolved now and I've just made a minor relesae 3.30.1. I think that we can merge now.

**Pull request merged**